### PR TITLE
The friendica constants have been moved to the app class

### PR DESCRIPTION
**Pull request by annando**
And some more constants moved.

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -58,6 +58,10 @@
  */
 class App
 {
+	const PLATFORM = 'Friendica';
```
Comment (tobiasd) [inline src/App.php:61]: @nupplaphil does this change affect the CI/build scripts somehow?

File: src/Module/Api/GNUSocial/GNUSocial/Config.php [hunk 2]
```diff
@@ -53,9 +53,9 @@ protected function rawContent(array $request = [])
 				'sslserver'    => null,
 				'ssl'          => DI::config()->get('system', 'ssl_policy') == App\BaseURL::SSL_POLICY_FULL ? 'always' : '0',
 				'friendica'    => [
-					'FRIENDICA_PLATFORM'    => FRIENDICA_PLATFORM,
-					'FRIENDICA_VERSION'     => FRIENDICA_VERSION,
-					'DB_UPDATE_VERSION'     => DB_UPDATE_VERSION,
+					'FRIENDICA_PLATFORM'    => App::PLATFORM,
+					'FRIENDICA_VERSION'     => App::VERSION,
+					'DB_UPDATE_VERSION'     => App::VERSION,
```
Comment (tobiasd) [inline src/Module/Api/GNUSocial/GNUSocial/Config.php]: Are these two really the same version?

Comment (annando) [reply src/Module/Api/GNUSocial/GNUSocial/Config.php]: No, this is a good catch.

Comment (annando) [reply src/Module/Api/GNUSocial/GNUSocial/Config.php]: Fixed

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Waiting for @nupplaphil to answer the question about CI.

File: src/App.php (hunk 1 above)
Comment (nupplaphil) [reply src/App.php:61]: No, I need the dbstructure.conf.php and the dbstructure.sql Version values, but not this one :-)

**Pull request merged**